rollup.config.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Created by arjun on 28/05/16.
+ */
+
+import babel from 'rollup-plugin-babel';
+
+export default {
+    entry: 'src/main.js',
+    format: 'es6',
+    // moduleName: 'HAMMER',
+    plugins: [ babel({exclude: 'node_modules/**'}) ],
+    dest: 'hammer.js',
+    intro: " (function(window, document, exportName, undefined) { \n'use strict' ",
+    outro: "})(window, document, 'Hammer');"
+
+};
```

Comment:
Missing newline, but also, is this still necessary now that there's the grunt build step?